### PR TITLE
Add back non-push version of function XrdAccAuthorizeObject

### PR DESCRIPTION
**Pull request by djw8605**
Want to make sure backwards compatibility continues.

### CODE REVIEW

File: src/XrdAccSciTokens.cc [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
 // The status-quo to retrieve the default object is to copy/paste the
 // linker definition and invoke directly.
+XrdVERSIONINFO(XrdAccAuthorizeObject, XrdAccSciTokens);
 extern XrdAccAuthorize *XrdAccDefaultAuthorizeObject(XrdSysLogger   *lp,
```
Comment (abh3) [inline src/XrdAccSciTokens.cc]: Could you get rid of this extern as it's not used anywhere (it's a holdover) and we eventually want to get rid of it.

**Review by abh3 — CHANGES_REQUESTED** [review]
Pretty much what has to be done. So, looks just fine. I am requesting one change that has nothing to do with your changes. It's removal of an unused extern.

**Comment by abh3** [comment]
Done. See comment.

**Comment by djw8605** [comment]
Removed extern.

**Pull request merged**